src/tests/ftest/erasurecode/rebuild_fio.py

```diff
@@ -29,30 +23,33 @@ def execution(self, rebuild_mode):
         Args:
             rebuild_mode: On-line or off-line rebuild mode
         """
+        read_option = self.params.get("rw_read", "/run/fio/test/read_write/*")
+
         # Kill last server rank first
-        self.rank_to_kill = self.server_count - 1
+        ranks_to_kill = [self.server_count - 1]
 
-        if 'on-line' in rebuild_mode:
-            # Enabled on-line rebuild for the test
-            self.set_online_rebuild = True
+        initial_usage = pool.pool_percentage_used()
+        print("initial pool usage = {}".format(initial_usage))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string